test/graphql.updateOrder.test.js

```diff
@@ -8,6 +8,7 @@ import models from '../server/models';
 const ordersData = utils.data('orders');
 
 const order = {
+  id: null,
```

Comment:
@eloyekunle do you know what was the breaking change in sequelize?

Reply:
In this case, we had a test saying return an error if there is no id. 

Sequelize v4 worked fine when `id` was undefined, but v5 gives errors so I needed to make it `null` to pass.